cinema/serializers.py

```diff
@@ -33,7 +33,16 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
Code Efficiency Issue: You've used `serializers.CharField` for the `image` field in the `MovieSerializer`. But as per the checklist, it's recommended to use `serializers.ImageField` for image fields. This will ensure that the field is properly validated as an image.